source/fundamentals/connection/tls.txt

```diff
@@ -285,9 +282,6 @@ variants the driver supports and the implications of using them.
            .build();
    MongoClientSettings settings = MongoClientSettings.builder()
            .applyToSslSettings(builder -> builder.enabled(true))
-           .streamFactoryFactory(NettyStreamFactoryFactory.builder()
-                   .sslContext(sslContext)
-                   .build())
            .build();
    MongoClient client = MongoClients.create(settings);
```

Comment:
For tech review: Can you confirm that this e.g. is still useful.

Reply:
Yes, it just has to be updated.